src/List.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+export default ({ ToDos }) => { // Export an annonymous function that takes in the ToDos we have and displays them in a list.
+
+    const clickHandler = (item) => {
+      item.completed = !item.completed;
+      const checkbox = document.getElementById(`${item.text}checkbox`);
```

Comment:
THere are actually much easier ways to do this than to selected the DOM. I would research how checkBoxes are handled in react most commonly.